National Library of Australia (new catalog).js

```diff
@@ -114,7 +109,19 @@ function scrapeMarc(text) {
 	var translator = Zotero.loadTranslator("import");
 	translator.setTranslator("a6ee60df-1ddc-4aae-bb25-45e0537be973");
 	translator.setString(text);
-	translator.translate();
+	translator.getTranslatorObject(function (obj) {
+		var record = new obj.record();
+		// Populate the record from the MARC
+		record.importBinary(text);
+		var item = new Zotero.Item();
+		// If there is a url for the digitised item add it
+		var url = record.getFieldSubfields("856");
+		if (url && url.length > 0 && url[0].u && url[0].z && url[0].z.startsWith("National Library of Australia digitised item")) {
+			item.url = url[0].u;
+		}
```

Comment:
`getFieldSubfields()` always returns an array (but it might be empty), so I think this could be simplified to:

```suggestion
		var [url] = record.getFieldSubfields("856");
		if (url && url.u && url.z && url.z.startsWith("National Library of Australia digitised item")) {
			item.url = url.u;
		}
```

Reply:
That's a neat trick to remember! Done.